cmd/di.go

```diff
@@ -266,7 +286,7 @@ func (di *Dependencies) bootstrapNodeComponents(nodeOptions node.Options) {
 	di.ConnectionRegistry = connection.NewRegistry()
 	di.ConnectionManager = connection.NewManager(
 		dialogFactory,
-		promiseIssuerFactory,
+		paymentIssuerFactory,
```

Comment:
why 'promiseIssuerFactory' is bad?

Reply:
mainly wanted to illustrate the fact that it's not only issuing promises as it will also have to keep track of balance, etc.